run.sh

```diff
@@ -65,14 +65,14 @@ done
 GALAXY_VIRTUAL_ENV="${GALAXY_VIRTUAL_ENV:-.venv}"
 if [ -d "$GALAXY_VIRTUAL_ENV" -a -z "$skip_venv" ];
 then
-    [ -n "$PYTHONPATH" ] && { echo 'Unsetting $PYTHONPATH'; unset PYTHONPATH; }
```

Comment:
@natefoo I've restored the old syntax except in one case (line 104), where I didn't like the use of backticks to quote inside the string, hope that's ok for you now.